Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1020,6 +1027,13 @@ extension BrowserViewController: BrowserToolbarDelegate {
         tabManager.selectedTab?.reload()
     }
 
+    func browserToolbarDidLongPressReload(browserToolbar: BrowserToolbarProtocol, button: UIButton) {
+        if #available(iOS 9.0, *) {
+            tabManager.selectedTab?.webView?.customUserAgent = "Macintosh; Max OS 10_10"
```

Comment:
So our iOS UA is:
Mozilla/5.0 (iPhone; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/**WebKitVersion** (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/**WebKitVersion**

For desktop, let's try this:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_1) AppleWebKit/**WebKitVersion** (KHTML, like Gecko) Safari/**WebKitVersion**

In other words, replacing the iPhone string with the Macintosh one and removing the FxiOS and Mobile parts of the UA. I'd put this in a static `desktopUserAgent` function in `UserAgent.swift`, right alongside our existing `defaultUserAgent`.